### DISCUSSION TITLE
In Production Environment, Can't open Dialog with Dialog Plugin

### DISCUSSION THREAD

**Discussion by Ethan0622** [quasarframework/quasar#12606] [General - Components / Directives / etc]
Hello everyone. I have encountered a strange thing.
My project have some dialog. I use the Dialog Plugins and create some components.
just like this (I use the <script setup> RFC):
![image](https://user-images.githubusercontent.com/67997980/155835110-04fd5468-1b0d-4ce5-8dff-3728e037eede.png)

then in homepage, I import this component and use $q.dialog to open it.
![image](https://user-images.githubusercontent.com/67997980/155835167-db26d582-78bf-4b67-8b6e-6f3d848ee151.png)

Obviously, It will work well. In Development Environment, I click the button and open the dialog successfully.
![image](https://user-images.githubusercontent.com/67997980/155835252-467c8c35-f390-455a-aa97-d43bce6dcee0.png)

Then, I use npm run build and put my project into My Server. I enter the index page, click the button to open dialog, but it failed.
some error in console.  I don't know what caused it. Maybe someone could help me?
![image](https://user-images.githubusercontent.com/67997980/155835532-fe4f90d6-4687-4755-8162-19fb33ce2977.png)

**Comment by yusufkandemir** [comment]
Please provide a repro. In this case, since it works on development, but doesn't work on production, I think a public GitHub repository would do.

**Reply by Ethan0622** [reply]
thanks, I create a simple public repository from my project. The URL is https://github.com/Ethan0622/quasar-dialog-plugin-production-discussion
I have updated some node_modules, but it still doesn't work on production.
